app/src/main/java/com/example/arc_exapmle/activity/MainActivity.kt

```diff
@@ -117,7 +124,13 @@ class MainActivity : AppCompatActivity() {
         when (item.itemId) {
 
             R.id.delete_all_notes -> {
-                mainActivityViewModel.deleteAllNotes()
+
+                CoroutineScope(Default).launch {
```

Comment:
chera bayad az to activity launch konim?
behtar nist view mon az inke ma darim on zir az coroutines estefade mikonim khabar nadashte bashe?

Reply:
hamchenin in launch scope dorosti fek nemikonam dashte bashe!
in scope jayi nadidam controll (cancel) beshe

Reply:
Hal shod